core/src/test/java/org/apache/iceberg/rest/RESTCatalogAdapter.java

```diff
@@ -474,6 +476,23 @@ public <T extends RESTResponse> T handleRequest(
           break;
         }
 
+      case REGISTER_VIEW:
+        {
+          if (null != asViewCatalog) {
+            LoadViewResponse response =
+                CatalogHandlers.registerView(
+                    asViewCatalog,
+                    namespaceFromPathVars(vars),
+                    castRequest(RegisterViewRequest.class, body));
+
+            responseHeaders.accept(
```

Comment:
We don't send ETags for views ATM

Reply:
Good catch. I have updated the PR now. 

I was following register table and didn't observe that we don't have ETag for views. Should we have a ticket to implement ETags for views?